test/role.test.js

```diff
@@ -93,11 +90,25 @@ describe('role model', function() {
 
   });
 
-  it('should automatically generate role id', function() {
+  it('should not allow duplicate role name', function(done) {
+    Role.create({ name: 'userRole' }, function(err, role) {
+      if (err) return done(err);
 
-    User.create({name: 'Raymond', email: 'x@y.com', password: 'foobar'}, function(err, user) {
-      // console.log('User: ', user.id);
-      Role.create({name: 'userRole'}, function(err, role) {
+      Role.create({ name: 'userRole' }, function(err, role) {
+        expect(err).to.exist; //jshint ignore:line
```

Comment:
@bajtos Do you know if there is a better way to fix this (besides ...to.exist())?